infrastructure/code/convert.go

```diff
@@ -466,15 +466,15 @@ func createAutofixWorkspaceEdit(filePath string, fixedSourceCode string) (edit s
 		NewText: fixedSourceCode,
 	}
 	edit.Changes = make(map[string][]snyk.TextEdit)
-	edit.Changes[filePath] = []snyk.TextEdit{singleTextEdit}
+	edit.Changes[absoluteFilePath] = []snyk.TextEdit{singleTextEdit}
 	return edit
 }
 
 // toAutofixSuggestionsIssues converts the HTTP json-first payload to the domain type
-func (s *AutofixResponse) toAutofixSuggestions(filePath string) (fixSuggestions []AutofixSuggestion) {
+func (s *AutofixResponse) toAutofixSuggestions(baseDir string, filePath string) (fixSuggestions []AutofixSuggestion) {
 	for _, suggestion := range s.AutofixSuggestions {
 		d := AutofixSuggestion{
-			AutofixEdit: createAutofixWorkspaceEdit(filePath, suggestion.Value),
+			AutofixEdit: createAutofixWorkspaceEdit(ToAbsolutePath(baseDir, filePath), suggestion.Value),
```

Comment:
Why create a delegate to the go package? Not an issue for me to approve, but looks unnecessary, and it's harder to see what happens, if it's hidden behind an indirection. 

![image](https://github.com/snyk/snyk-ls/assets/20150761/605ebd83-21c4-4414-9bca-a5c3a5c2023d)